app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
`SALARY_FREQUENCIES = %w[year month fourweeks week].freeze` is also in `models/wpcc/your_details_form.rb`. Can it be removed from there? I was just about to add this comment.

I was thinking along the same lines as @willhall88. It strikes me that some of these constants could be environment vars and should we be more consistent in terms of how / where we store them? e.g. in a yaml file?